Wrappers/Python/cil/optimisation/algorithms/CGLS.py

```diff
@@ -110,7 +125,7 @@ def update(self):
         #self.p = self.s + self.beta * self.p
         self.p.sapyb(self.beta, self.s, 1, out=self.p)
 
-        self.normx = self.x.norm()
+        self.normx = self.x.norm()#Can be deprecated with tolerance 
```

Comment:
It migth be more obvious to change this to `TODO: Deprecated, remove when CGLS tolerance is removed`

It should also be added to a new issue akin to https://github.com/TomographicImaging/CIL/issues/1443 stating which version it was marked as deprecated in and suggesting removal for v25 (or 26?)

Reply:
Done